scraper/src/libretexts2zim/processor.py

```diff
@@ -379,3 +383,31 @@ def run(self) -> Path:
                 )
 
         return zim_path
+
+    def _process_css(
+        self,
+        creator: Creator,
+        target_filename: str,
+        css_location: str,
+        css_content: str | bytes | None = None,
+    ):
+        """Process a given CSS stylesheet
+        Download content if necessary, rewrite CSS and add CSS to ZIM
+        """
+        if not css_location:
+            raise ValueError(f"Cannot process empty css_location for {target_filename}")
+        if not css_content:
+            css_buffer = BytesIO()
+            stream_file(css_location, byte_stream=css_buffer)
+            css_content = css_buffer.getvalue()
+        url_rewriter = ArticleUrlRewriter(
+            article_url=HttpUrl(css_location),
+            article_path=ZimPath(target_filename),
+        )
+        css_rewriter = CssRewriter(url_rewriter=url_rewriter, base_href=None)
+        result = css_rewriter.rewrite(content=css_content)
+        self.items_to_download = {
```

Comment:
Please use the `update` method for this update. There's no need to recreate the dict everytime

Reply:
`update` will replace existing keys ; I wanted to ensure we keep already existing entries. I should have add a comment. I want to keep existing keys values because we might have "overlapping" ZimPath (two URLs leading to same ZimPath) and it seemed to me that first URL encountered is maybe better than the next one. I'm not sure about this argument anymore, and it even seems better to keep all URLs, so that if one fails we can try the next ones.

Reply:
comment seems mandatory then